server/notification-providers/whapi.js

```diff
@@ -0,0 +1,46 @@
+const NotificationProvider = require("./notification-provider");
+const axios = require("axios");
+
+class Whapi extends NotificationProvider {
+
+    name = "whapi";
+
+    /**
+     * @inheritdoc
+     */
+    async send(notification, msg, monitorJSON = null, heartbeatJSON = null) {
+
+        let okMsg = "Sent Successfully.";
+
+        let apiUrl = notification.whapiApiUrl;
+        let apiToken = notification.whapiAuthToken;
+        let toNumber = notification.whapiToNumber;
+
+        try {
+
+            let config = {
+                headers: {
+                    "Accept": "application/json",
+                    "Content-Type": "application/json",
+                    "Authorization": "Bearer " + apiToken,
+                }
+            };
+
+            let data = {
+                "to": toNumber + "@s.whatsapp.net",
+                "body": msg,
+            };
```

Comment:
Looking at the api, I think this might be an interesting setting to add.
Opinions?

![image](https://github.com/louislam/uptime-kuma/assets/26258709/b11c6134-7d55-43b3-9666-53644b7970a1)

Reply:
The **mentions** parameter is only useful when sending in a group. The problem is that you still need to write "@PHONENUMBER" in the text message or Whatsapp will behave a bit wierdly, showing you that you got mentioned while not showing anything particular in the text message. The only reason this could be useful is because it bypasses muted notifications, but I think that you shouldn't mute a bot that does uptime checks :sweat_smile: